common-lib/src/main/scala/com/gu/mediaservice/lib/metadata/ImageMetadataConverter.scala

```diff
@@ -28,10 +45,12 @@ object ImageMetadataConverter {
       subLocation         = fileMetadata.iptc.get("Sub-location"),
       city                = fileMetadata.iptc.get("City"),
       state               = fileMetadata.iptc.get("Province/State"),
-      country             = fileMetadata.iptc.get("Country/Primary Location Name")
+      country             = fileMetadata.iptc.get("Country/Primary Location Name"),
+      suppliersCategory   = fileMetadata.iptc.get("Category")
```

Comment:
The standards & codes for "Supplemental Category" are just a horrible eye bleeding mess which i'm not going to touch with a very long stick.